File: Practice(Beginner)/Python3/Palindromic_Substrings.py
# Source: https://www.codechef.com/problems/STRPALIN


def isPalindromePossible(s1, s2):
    for i in s1:
        if(i in s2):
            return True
    return False


t = int(input())

for test in range(t):
    s1 = input()
    s2 = input()
    print("Yes") if(isPalindromePossible(s1, s2)) else print("No")

File: Practice(Beginner)/C++/Palindromic substrings.cpp
// Source: https://www.codechef.com/problems/STRPALIN

#include <bits/stdc++.h>
using namespace std;
int main(){
    int t;
    cin >> t;
    while (t--){
        string s1;
        string s2;
        cin >> s1 >> s2;
        bool flag = false;
        for (int i = 0; i < s1.size(); ++i) {
            for (int j = 0; j < s2.size(); ++j) {
                if(s1[i] == s2[j]){
                    flag = true;
                    break;
                }
            }
            if(flag){
                break;
            }
        }
        flag ? cout << "Yes" << endl : cout << "No" << endl;
    }
}